src/components/Home.jsx

```diff
@@ -1,24 +1,40 @@
 import React, { useState, useEffect } from "react";
+import Filter from "./Filter";
 import ArticleList from "./ArticleList";
 
 import { getArticles } from "../api/requests";
 
 const Home = () => {
   const [articles, setArticles] = useState([]);
+  const [selectedTopic, setSelectedTopic] = useState("");
+  const [filteredArticles, setFilteredArticles] = useState([]);
 
   useEffect(() => {
     getArticles()
       .then(({ data }) => {
         setArticles(data.articles);
-        console.log(data.articles);
       })
       .catch((err) => console.log(err));
   }, []);
 
+  useEffect(() => {
+    const newFilteredArticles = articles.filter((article) => {
+      return article.topic.includes(selectedTopic);
+    });
```

Comment:
You should be filtering using your API request and query parameters instead of locally filtering through all the articles yourself. Axios allows you to add params as a extra option when you make requests.

Reply:
Ah ok, will sort thanks